### ISSUE TITLE
More-info on entity card and others dont hide attributes

### ISSUE THREAD

**Issue by Mariusthvdb** [Mariusthvdb/custom-ui#39]
never noticed before but clicking an entity on an `type: entity` card doesn't respect the settings we set under hidden attributes. As on an custom:mini-graph-card.

tagging @CAB426 since you fixed this on the more-info before, please could you once more have a look?

eg:
Lovelace card:
```
  - type: custom:mini-graph-card
    entities:
      - entity: sensor.co2_living
        name: Co2
      - entity: sensor.temperatuur_living
        name: Temp Living
      - entity: sensor.luchtvochtigheid_living
        name: Vocht
      - entity: sensor.schimmel_sensor
        name: Schimmel
    height: 100
    line_width: 4
    font_size: 75
    show:
      graph: true
      fill: true
      extrema: true
```
clicking the entity in the card results in:

<img width="392" alt="Schermafbeelding 2021-05-16 om 23 23 23" src="https://user-images.githubusercontent.com/33354141/118413090-c5d31d80-b69d-11eb-863f-529d310d8053.png">

while the same entity is correctly treated in a entities or glance card:

<img width="397" alt="Schermafbeelding 2021-05-16 om 23 23 33" src="https://user-images.githubusercontent.com/33354141/118413115-e8fdcd00-b69d-11eb-8049-d45445af884c.png">

this happens on the latest version 20210511, but also on earlier versions. I guess the Lovelace cards use an other way of calling the more info on these cards, which I hope can be added to the `updateMoreInfo()` function in custom-ui.

Thanks if you could have a look

**Comment by Mariusthvdb** [comment]
got this help from Thomas Loved on [Discord](https://discord.com/channels/330944238910963714/351047592588869643/843819957308751912):

> I'd guess you're seeing timing differences.
> What you need to do is move one step at a time in that chain, and whenever there's a shadowRoot you check if the parent element has a updateComplete property, and if so await that before continuing.
> Here's a function I keep reusing for this sort of stuff: https://github.com/thomasloven/lovelace-card-tools/blob/master/src/helpers.js

**Comment by Mariusthvdb** [comment]
fixed in 2021.5.28
https://github.com/Mariusthvdb/custom-ui/releases/tag/2021.5.28
